.github/workflows/test.yml

```diff
@@ -204,6 +204,18 @@ jobs:
           path: frontend
           fetch-depth: 1
 
+      - name: Wait for Backend
+        run: |
+          npm install -g wait-on && \
+          wait-on http://localhost:3000/api/v3/health --timeout 200000
+        # run: |
```

Comment:
Maybe the commented part should be removed completely.

Reply:
Yes I will remove them, I need to re-run this test more times to see if it is stable

Reply:
Yes I will remove them, I need to re-run this test more times to see if it is stable